.github/workflows/ci.yml

```diff
@@ -86,4 +164,5 @@ jobs:
           bats-version: 1.2.1
```

Comment:
@Zanda256  Can you remove ci.yaml from this PR? It needs to get rid of BATS tests and this issue has already been assigned to someone. So I think you should delete ci.yaml from this PR